src/elin_bjoernson_ex/ex01/letter_counts.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+def letter_freq(txt):
+    txt = txt.lower()  # make all letters lowercase
+    txt = sorted(txt)  # sort alphabetically
```

Comment:
Nice thinking.

> As an extra challenge, modify the code **printing the table** so that letters are printed in alphabetical order

`sorted` function should be in the main code, not here.
`for letter, count in sorted(frequencies.items()):`